deps/build.jl

```diff
@@ -22,6 +22,8 @@
     push!(dependencies, "build_FLINT.v2.6.2.jl")
   elseif flint_ver == v"2.6.3+0" || flint_ver == v"2.6.3+1"
     push!(dependencies, "build_FLINT.v2.6.3.jl")
+  elseif flint_ver == v"200.690.0+0"
+    push!(dependencies, "build_FLINT.v200.690.0.jl")
```

Comment:
Actually, why do we need this `deps/build.jl` at all, and its used of BinaryProvider? I thought that's what we had LoadFlint.jl for?

Reply:
Nemo loads arb and antic, which need the `build_FLINT_*.jl`.

Reply:
OK, good point! (I mean, I am still pretty confident that those could be made to use `LoadFlint.jl`; but it's non-trivial work, and just not worth the effort just for Julia 1.0-1.2, so no point in debating this further.)

Reply:
You are right. I will give it a try today.

Reply:
From my point of view, that would be a waste of your valuable time -- if it works this way, whatever, it is good enough; hopefully one day we'll drop support for Julia before 1.3, and then this will all go anyway. No point in polishing a turd...

Alas, do what you feel you must do ;-).

Reply:
Does not seem to be as easy as picking the `build_FLINT_*` from LoadFlint. I won't pursue this further.